system-int-test/test/cmr/system_int_test/search/granule_shapefile_search_test.clj

```diff
@@ -148,7 +148,7 @@
         "test_box_with_holes.zip" [whole-world very-wide-cart richmond]
 
         "Polygon across the antimeridian"
-        "antimeridian.zip" [whole-world very-tall-cart across-am-poly across-am-br am-point]
+        "antimeridian.zip" [whole-world very-wide-cart across-am-poly across-am-br am-point]
```

Comment:
Why did this behavior change? Are you sure that this is finding the correct granules now?

Reply:
I changed the contents of the shapefile to match what I was using with the GeoJSON test. The results are correct.

Reply:
I changed the contents of the shapefile to match what I was using with the GeoJSON test. The results are correct.